### PR TITLE
Add VirSorter package

### PR DESCRIPTION
**Pull request by wclose**
This pull request is to add the VirSorter program to bioconda.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/virsorter/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  sha256: {{ virsorterSha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - perl-bioperl
```
Comment (bgruening) [inline recipes/virsorter/meta.yaml]: you probably want to have perl here as well

File: recipes/virsorter/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  run:
+    - blast
```
Comment (bgruening) [inline recipes/virsorter/meta.yaml]: please add perl as well

File: recipes/virsorter/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    - diamond =0.9.14
```
Comment (bgruening) [inline recipes/virsorter/meta.yaml]: do you need the strict pinning?

**Comment by wclose** [comment]
@bgruening I looked into it a bit more and it turns out the incompatible packages are all being installed due to using `perl-bioperl` so I don't think they are supposed to be mutually exclusive. They work fine in a normal environment but not in the build environment. Any ideas on how to fix it/avoid it?

File: recipes/virsorter/meta.yaml (hunk 1 above)
Comment (wclose) [reply recipes/virsorter/meta.yaml]: done

File: recipes/virsorter/meta.yaml (hunk 2 above)
Comment (wclose) [reply recipes/virsorter/meta.yaml]: done

File: recipes/virsorter/meta.yaml (hunk 3 above)
Comment (wclose) [reply recipes/virsorter/meta.yaml]: The software developer has them pinned at these versions when using the software on other servers so I want to maintain the settings for comparability.

**Comment by dpryan79** [comment]
@bioconda-bot merge

**Pull request merged**